dist/c2fo-ui-components.css

```diff
@@ -0,0 +1,45 @@
+.pager {
```

Comment:
Not sure if I fully agree with moving a piece that makes up the whole of a module away from it's markup and behavior.  Having a style-base for a component is important.  

The ui-guide is only concerned about how the c2fo project implements these things and that shouldn't override predictable behavior to other projects wanting to use this repo, nor should it impose style duplication for projects other than the c2fo supplier/buyer app paradigm.

No need to debate here - we can discuss in the chapter meetings when it comes up but just wanted to log a point for discussion when the ui-guide gets further along.